tsc/process/deprecation.md

```diff
@@ -0,0 +1,13 @@
+**Deprecation Strategy for OpenVDB**
+
+OpenVDB is committed to supporting the current and last two years of the VFX Reference Platform.
+
+For example, as of writing it is March 2019 which means VFX Reference Platform 2019, 2018 and 2017
+
+This infers the following support:
+
+* OpenVDB ABI=4, ABI=5 and ABI=6
+* Houdini 16.5, 17.0 and 17.5
+* C++11 and C++14
+
+In January 2020, OpenVDB will drop support for VFX Reference Platform 2017. This will mean deprecating support for Houdini 16.5, ABI=4 and C++11.
```

Comment:
Will we be deprecating that support, or eliminating it?

Reply:
Eliminating is better, fixed!